### ISSUE TITLE
API to determine if content is non-conforming

### ISSUE THREAD

**Issue by tciuro** [faceleg/ACEView#50]
Hello,

I haven't seen an `ACEView/ACEViewDelegate` API to determine if the content of the view are not conforming to the specified language. For example, if I set the `ACEView` mode to be `ACEModeJSON` and I edit the contents removing a comma, a red "X" appears in the gutter (awesome!). I'd love to programmatically figure out if the view is showing one or more "X"'s so that I can redraw the rest of the UI accordingly. Did I overlook this API somewhere? Thanks!

**Comment by faceleg** [comment]
Is there a way to determine this with the ACE editor?

**Comment by tciuro** [comment]
@faceleg  I don't understand your question. What do you mean with "ACE editor"?  I've checked the public APIs and I don't see how I could gain access.

**Comment by faceleg** [comment]
I mean the ACE HTML editor itself

**Comment by faceleg** [comment]
@tciuro did my explanation make more senese?

**Comment by tciuro** [comment]
Yes, thank you. I posted a question in the forum hoping to figure out whether it's possible.
